src/Wrapper.js

```diff
@@ -178,11 +178,19 @@ class Wrapper extends React.PureComponent {
       }
     })
   }
+
   isAppInstalled(instanceId) {
-    const { apps } = this.props
     return (
       staticApps.has(instanceId) &&
-      !!apps.find(app => addressesEqual(app.proxyAddress, instanceId))
+      Boolean(this.getAppByProxyAddress(instanceId))
+    )
+  }
+
+  getAppByProxyAddress = proxyAddress => {
```

Comment:
What do you think about pulling this out and making a curried version that can be imported anywhere, i.e. `  getAppByProxyAddress = apps => proxyAddress => {...
`

This should allow us to reduce the size of wrapper.

Reply:
Yes let’s do this, `Wrapper` is large enough already! Adding this also made me think that it could be nice to move the loaded apps in a React context.